app/forms/questionnaire_form.py

```diff
@@ -28,26 +34,85 @@ def validate(self):
         Validate this form as usual and check for any form-level date-range validation errors
         :return:
         """
-        valid_form = True
-        valid_fields = FlaskForm.validate(self)
+        if not FlaskForm.validate(self):
+            return False
```

Comment:
This isn't idea as it doesn't show all of the errors the 1st time around. Which can be annoying for the respondent.

Reply:
I think in the previous version respondent only sees field errors first anyways, will double check

Reply:
on testing it returns both field and form validation. @nealedj what sort of behaviour do we want?

Reply:
On PreProd it returns both straight away.
Run `test_dates.json`, enter a from date after the to date, don't answer any other questions and you will see field errors and range errors.

Reply:
It has always been field validations first.

Reply:
Yes, but with this change, if the range is invalid and another question has a field error then the range validation doesn't show.

Reply:
ok have reverted back to valid fields notion, so exerts same behaviour as before